src/templates/infra/data/database/postgres/migration.ejs

```diff
@@ -1,17 +1,44 @@
-
 exports.up = async function (knex) {
-    knex.schema.hasTable('<%- props.table %>')
-        .then(function (exists) {
-            if (exists) return
-            return knex.schema
-                .createTable('<%- props.table %>', function (table) {<% for(colum of props.columns) { %>
-                    <%- colum %><% } %>
-                    table.timestamps()
-                })
-        })
+    return Promise.all([
+        knex.schema.hasTable('<%- props.table %>')
+            .then(function (exists) {
+                if (exists) return
+                return knex.schema
+                    .createTable('<%- props.table %>', function (table) {
+                        <% for(colum of props.columns) {%> 
+                        <%- colum %>  <%}%>
+                        <%if (props.ref) { 
+                            props.ref.forEach(function(link){ 
+                                if (link.relationship === 'One-to-One') { %>
+                        // table.integer('<%-link.columnName %>').unsigned().nullable()
```

Comment:
this comment is necessary?

Reply:
the idea are to not create a migration that does not work, due to not has an order of migrations when we write the files. These relationships throw errors due the dependence dont exist yet ...